app/controllers/admin/publishers_controller.rb

```diff
@@ -107,6 +107,16 @@ def refresh_uphold
     redirect_to admin_publisher_path(@publisher.id)
   end
 
+  def sign_in_as_user
```

Comment:
I don't like this, because it takes you out of the context of being an admin. Admins could login and then be under the user's account for up to 30 days. 

I would much rather have the home page use a singular instance variable for `@publisher`, and then be able to access the `publishers_controller` as an administrator and view the publishers account that way.

Reply:
Taking this offline, as I'll ask support what their thoughts are. The standard convention is found here: https://github.com/plataformatec/devise/wiki/How-To:-Sign-in-as-another-user-if-you-are-an-admin